1724832275961	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1724856464860	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1724856464860	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1724856464861	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1724856464863	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1724856464876	Sync.SyncScheduler	DEBUG	Woke from sleep.
1724856464876	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1724856464876	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1724856464877	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1724856466930	Sync.Service	DEBUG	User-Agent: Firefox/129.0.1 (Linux x86_64) FxSync/1.131.0.20240812083845.desktop
1724856466930	FirefoxAccounts	INFO	Polling device commands.
1724856466931	Sync.Service	INFO	Starting sync at 2024-08-28 09:47:46 in browser session rCsbPfhqXnhY
1724856466931	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1724856466933	Sync.Status	INFO	Resetting Status.
1724856466933	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1724856466934	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1724856466934	FirefoxAccounts	DEBUG	getOAuthToken enter
1724856466934	FirefoxAccounts	TRACE	getCachedToken returning cached token
1724856466934	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1724856466934	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1724856466935	Sync.SyncAuthManager	INFO	Getting sync key
1724856466935	FirefoxAccounts	DEBUG	getOAuthToken enter
1724856466935	FirefoxAccounts	TRACE	getCachedToken returning cached token
1724856466935	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1724856466935	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1724856466935	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1724856466935	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1724856466935	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1724856466937	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/device/commands?index=0
1724856466938	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1724856466938	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1724856466939	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1724856466939	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1724856466939	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1724856466939	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1724856466939	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1724856466939	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1724856466940	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1724856466940	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1724856466943	Services.Common.RESTRequest	DEBUG	GET https://api.accounts.firefox.com/v1/account/device/commands?index=0 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1724856466943	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.sys.mjs:478
1724856466943	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
request()@resource://services-common/hawkclient.sys.mjs:230
1724856466943	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 630"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:630
request()@resource://services-common/hawkclient.sys.mjs:238
1724856466943	Hawk	DEBUG	(Response) /account/device/commands?index=0: code: null - Status text: null
1724856466943	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:246
1724856466943	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:247
1724856466943	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 630"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:630
_constructError()@resource://services-common/hawkclient.sys.mjs:113
request()@resource://services-common/hawkclient.sys.mjs:252
1724856466944	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
_constructError()@resource://services-common/hawkclient.sys.mjs:114
request()@resource://services-common/hawkclient.sys.mjs:252
1724856466944	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 615"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:615
_constructError()@resource://services-common/hawkclient.sys.mjs:115
request()@resource://services-common/hawkclient.sys.mjs:252
1724856466944	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 662"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:662
_constructError()@resource://services-common/hawkclient.sys.mjs:121
request()@resource://services-common/hawkclient.sys.mjs:252
1724856466944	FirefoxAccounts	ERROR	error GETing /account/device/commands?index=0: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]","message":null,"code":null,"errno":null}
1724856466944	Sync.SyncScheduler	ERROR	Fetching missed remote commands failed.: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]","message":null,"code":null,"errno":null}
1724856466944	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/214229003/info/collections?v=1.131.0 failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1724856466944	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1724856466944	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1724856466944	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1724856466945	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1724856466945	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1724856466948	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
